docs/guides/external-node/10_attestator_committee.md

```diff
@@ -0,0 +1,41 @@
+# Attester Committee
+
+## Overview
+
+Attester committee is a subset of ZKSync nodes. 
+After each l1 batch execution, participating nodes sign the execution result and send it back to the main node.
+
+The ultimate goal is to make L1 commit operation contingent on such signatures.
+This will improve the security and finality guarantees: 
+having these signatures on L1 shows that additional actors executed the corresponding blocks - 
+and ended up with the same state root hash.
+
+## Current State
+
+Before we start the L1 integration,
+we want to make sure we are able to consistently reach the quorum and collect the signatures in a timely manner. 
+Currently the main node just stores the signatures in the local DB (`l1_batches_consensus` table).
+
+We run a (temporary) PoA network of attesters where the Main Node administrator defines the committee for every L1 batch.
+We'll be developing incentives and potential tokenomics later on - please stay tuned.
+
+We are looking for participants to this network.
+
+## Participating in the Attester Committee
+
+Joining the attester committee imposes no additional computational, operational, security, or business overhead for EN operators.
+
+The only difference in behaviour is that the node would asynchronously sign batches and send those signatures to the main node.
+Node checks if its public key is part of the committee for the current l1 batch - if it is, this logic kicks in.
+We expect the participating nodes to have very high uptime, 
+but there are no penalties for not submitting votes (and we wouldn't have any way to impose this).
+
+One can leave the committee any time.
+
+The only action that is required to participate is to share your attester public key with the Main Node operator.
```

Comment:
How should they share their key? If it's by Slack we should specify that.

Reply:
Slack is not open for everyone so it's kind of purposefully vague.. If someone from outside wants to participate they could open a GH discussion. 

I'll keep as is - but if you have a concrete proposal feel free to add a suggestion

Reply:
We can tell them to open an issue on zksync-era with title "Attester key submission", or to use any other communication channel that is convenient for them.